test/unit/middlewares/tracing.spec.js

```diff
@@ -1078,9 +1080,10 @@ describe("Test TracingMiddleware localAction", () => {
 							retries: 3,
 							timeout: 5
 						},
+						requestID: "request-id",
 						remoteCall: false
 					},
-					traceID: "tracer-trace-id",
```

Comment:
Why is it also changed?

Reply:
When I originally did the global action/event tags I didn't realize that you were setting the requestID to null to test if `getCurrentTraceID` was called. Since the the tests for the global action/event tags have nothing to do with this behavior I thought this was a good opportunity to clean up these tests to reduce the number of assertions in them.